package.json

```diff
@@ -44,12 +47,22 @@
     "babel-preset-stage-2": "^6.24.1",
     "chai": "^4.1.2",
     "chai-as-promised": "^7.1.1",
+    "eslint": "^5.7.0",
```

Comment:
This is giving an `incompatible engine` error on CI.  Can you switch the circle.yml to use `npm` for the install rather than `yarn`?